diagnostic.md

```diff
@@ -38,7 +47,12 @@ $base-margin: 1.5em;
 ```
 
 ```scss
-// your response here
+
+li {
+  margin: $new-margin;
+}
+
+$new-margin: ($base-margin * 0.5);
 ```
```

Comment:
Same as above. Make sure you define `$new-margin` before using it for the `li`'s styles.